pipelines/base/config-dump.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: dump-config
+spec:
+  params:
+  - name: REPO_NAME
+    type: string
+    default: .github
+  - name: BRANCH_NAME
+    type: string
+    default: add-initial-peribolos-config
+  - name: GIT_EMAIL
+    type: string
+  - name: GIT_NAME
+    type: string
+  workspaces:
+  - name: secret-data
+    description: location for installation secret
+  - name: shared-data
+    description: location for passing default branch name to the second step
```

Comment:
This is meant just for passing data between steps, correct? Then rather use an `emptyDir` volume, mounted to both steps

https://tekton.dev/docs/pipelines/tasks/#specifying-volumes

Like for example here:
https://github.com/tektoncd/catalog/blob/f4708d478ee8fac6b5b68347cde087cb7c1d1b1c/task/s2i/0.2/s2i.yaml#L86-L88